tests/test_ndarrays.py

```diff
@@ -46,6 +49,36 @@ def test_polygon(self):
             [(1.0, 3.0), (1.0, 4.0), (2.0, 4.0), (2.0, 3.0), (1.0, 3.0)]
         )
 
+    @unittest.skipIf(not numpy, 'numpy not installed')
+    def test_multipoint(self):
+        arr = numpy.array([[1.0, 1.0, 2.0, 2.0, 1.0], [3.0, 4.0, 4.0, 3.0, 3.0]])
```

Comment:
Also here, copied the tests above but adapted using the normal constructor, to ensure we keep tests for this functionality when the adapter tests are removed.